### PR TITLE
First pass at Connector Example

### PR DESCRIPTION
**Pull request by ggam**

### CODE REVIEW

**Comment by ggam** [comment]
This is a first pass so I left "EJB" on code samples. Replacing it requires coordination with the examples project code.

File: src/main/jbake/content/connectorexample002.adoc [hunk 1]
```diff
@@ -50,7 +50,7 @@ provides and implements a custom client interface for Jakarta EE
 applications to use. This interface is simpler than the Common Client
 Interface (CCI).
 
-The `trading-war` module is a web application with a JavaServer Faces
+The `trading-war` module is a web application with a Jakarta Faces
```
Comment (bshannon) [inline src/main/jbake/content/connectorexample002.adoc]: Jakarta Server Faces

File: src/main/jbake/content/connectorexample003.adoc [hunk 2]
```diff
@@ -57,8 +57,8 @@ a traffic update looks like this:
  ]}
 ----
 
-The `traffic-rar` module implements the inbound contract of the Jakarta EE
-Connector Architecture. This module subscribes to the traffic
+The `traffic-rar` module implements Jakarta Connectors the inbound contract.
```
Comment (bshannon) [inline src/main/jbake/content/connectorexample003.adoc]: Should be:

The `traffic-rar` module implements the inbound contract of the Jakarta Connectors specification.

File: src/main/jbake/content/connectorexample003.adoc [hunk 3]
```diff
@@ -90,7 +90,7 @@ adapter.
 adapter.
 
 The business interface defined by the resource adapter is not specified
-in the Jakarta EE Connector Architecture; it is specific to the EIS.
+in the Jakarta Connectors spec; it is specific to the EIS.
```
Comment (bshannon) [inline src/main/jbake/content/connectorexample003.adoc]: "spec" -> "specification"

**Pull request merged**